service-deployment/src/grafana/pai-nodeview-dashboard.json.template

```diff
@@ -493,7 +401,7 @@
           "thresholds": [],
           "timeFrom": null,
           "timeShift": null,
-          "title": "$node Disk Space Used",
+          "title": "Disk Space Used",
```

Comment:
Is it really 'disk space used'? I found the page calls promethus api to get disk read/write rates.

Reply:
Changed title into "Disk"